src/SFML/Window/DRM/InputImplUDev.cpp

```diff
@@ -564,7 +564,7 @@ bool InputImpl::isKeyPressed(Keyboard::Key key)
         return false;
 
     update();
-    return keyMap[key];
+    return keyMap[static_cast<size_t>(key)];
```

Comment:
```suggestion
    return keyMap[static_cast<std::size_t>(key)];
```

Reply:
Done, should I also replace the occurrences in line 304 - 305?